zadanie 19.1.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+<html lang="pl">
+<head>
+    <meta charset="UTF-8">
+    <title>Rejestracja</title>
+    <style>
+    [id=city]{ width:40%; float:left}
+    [id=voivodeship]{ width:45%; float:left}
+    [id=ZIP code]{ width:50%; float:left}

```

Comment:
Jeżeli odwołujesz się do identyfikatora, to stosuj selektor identyfikatora:
https://app.javastart.pl/kurs/technologie-webowe/web-css-selektory/lekcja/css-selektor-klasy-identyfikatora
a nie atrybutu. Czyli bez kwadratowych nawiasów, tylko zamiast tego ze znakiem #.